winui/Cartesian-Charts/Legend.md

```diff
@@ -378,103 +378,18 @@ By clicking on disabled legend item, we can view the associated `series`,
-
-### Docking the Legend Position
-
-By using the [DockPosition](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartLegend.html#Syncfusion_UI_Xaml_Charts_ChartLegend_DockPosition) property, legends can be docked to the left, right, and top or bottom of the chart area. By default, the chart legend is docked at the top of the chart as mentioned earlier.
-
-To display the legend at the bottom, you can set the [DockPosition](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartLegend.html#Syncfusion_UI_Xaml_Charts_ChartLegend_DockPosition) as `Bottom` as shown in the following code sample.
-
-{% tabs %}
-
-{% highlight xaml %}
-
-<chart:SfCartesianChart.Legend>
-    <chart:ChartLegend DockPosition="Bottom"/>
-</chart:SfCartesianChart.Legend>
-
-{% endhighlight %}
-
-{% highlight c# %}
-
-chart.Legend = new ChartLegend()
-{
-   DockPosition = ChartDock.Bottom
-};
-
-{% endhighlight %}
-
-{% endtabs %}
-
-![Positioning the legend at right in WinUI Chart](Legend_images/WinUI_chart_legend_dockPosition.png)
-
-### Floating Legends
-
-To position the legend at any arbitrary location within the chart, you need to set [DockPosition](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartLegend.html#Syncfusion_UI_Xaml_Charts_ChartLegend_DockPosition) as `Floating` and provide its relative position by using the [OffsetX](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartLegend.html#Syncfusion_UI_Xaml_Charts_ChartLegend_OffsetX) and [OffsetY](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartLegend.html#Syncfusion_UI_Xaml_Charts_ChartLegend_OffsetY) properties.
-
-{% tabs %}
-
-{% highlight xaml %}
-
-<chart:SfCartesianChart.Legend>
-    <chart:ChartLegend DockPosition="Floating" Orientation="Vertical" OffsetX="30" OffsetY="45" ItemMargin="3"/>
-</chart:SfCartesianChart.Legend>
-
-{% endhighlight %}
-
-{% highlight c# %}
-
-chart.Legend = new ChartLegend()
-{
-    DockPosition = ChartDock.Floating,
-    Orientation = ChartOrientation.Vertical,
-    OffsetX = 30,
-    OffsetY = 45,
-    ItemMargin = new Thickness(3)
-};
-
-{% endhighlight %}
-
-{% endtabs %}
-
-![Floating legend support in WinUI Chart](Legend_images/WinUI_chart_legend_floating.png)
-
-## Orientation
-
-Orientation of the legend items can be aligned vertically or horizontally by setting the [Orientation](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartLegend.html#Syncfusion_UI_Xaml_Charts_ChartLegend_Orientation) property of legend. By default, the value of [Orientation](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartLegend.html#Syncfusion_UI_Xaml_Charts_ChartLegend_Orientation) property is [Horizontal](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartOrientation.html#Syncfusion_UI_Xaml_Charts_ChartOrientation_Horizontal).
+To display the legend at the left, set the [Placement]() as [Left]() as in below code snippet.
```

Comment:
To display the legend at the left, set the [Placement]() as [Left]() as in the following code sample.